src/spaceObjects/scanProbe.cpp

```diff
@@ -40,7 +49,8 @@ REGISTER_SCRIPT_SUBCLASS_NO_CREATE(ScanProbe, SpaceObject)
 
 REGISTER_MULTIPLAYER_CLASS(ScanProbe, "ScanProbe");
 ScanProbe::ScanProbe()
-: SpaceObject(100, "ScanProbe")
+: SpaceObject(100, "ScanProbe"),
+  probe_speed(1000.0f)
```

Comment:
probe_speed better be replicated now as well, or clients will simulate that wrong and probes with a non-default speed will look janky.